debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleConnectorConfig.java

```diff
@@ -1958,6 +1987,16 @@ public Integer getOpenLogReplicatorPort() {
         return openLogReplicatorPort;
     }
 
+    /**
+     *
+     */
+    public String getLogMiningPathToDictionary() { return logMiningPathToDictionary; }
```

Comment:
Add documentation and please use new lines for the return and brackets like other methods.